src/redux/reducers/index.js

```diff
@@ -230,6 +243,22 @@ export const collectionsReducer = (state = preloadCollections, action) => {
         draft[collectionId].showMarkedOnly = showMarkedOnly;
       });
     }
+    case actionTypes.TOGGLE_SORTING_STATE: {
+      const { collectionId, sortField } = action.payload;
+      return produce(state, (draft) => {
+        if (state[collectionId].savedRecordsSortingState.activeSortField === sortField) {
```

Comment:
minor, but: aren't most of the things in collections state "saved" ?
(the key thing differentiating this is that it is for the "preview" screen, or whatever it's called, now?)

Reply:
this piece of state can be renamed to `DetailsPanelSortingState`

Reply:
oh, this is actually for sorting "saved" records (only?) -- not for highlight,  or other records that are not "saved" ?

Reply:
correct. this only applies to the records rendered under the `SortingHeader` aka the "saved" records.